openhtf/io/frontend/src/app/station.header.ts

```diff
@@ -0,0 +1,34 @@
+// Copyright 2016 Google Inc.All Rights Reserved.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+
+//     http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+import {Component, Input} from 'angular2/core';
+import {NumberPipe}       from 'angular2/common';
+
+import {Countdown} from './utils';
+
+import 'file?name=/templates/station.header.html!./station.header.html';
```

Comment:
@fahhem I see now, you're asking about line 21 in particular, the template import. These 'file' loader imports are so webpack can put the files into directories that will be served by the static file handler, at the same time as allowing their sources to live side-by-side with the components that use them.

I have no idea what I'm doing when it comes to webpack; I'm learning as I go. So if there's a more proper way to do this I'm all ears!